src/components/WorkspaceAdd/WorkspaceAddButton.test.tsx

```diff
@@ -40,12 +40,10 @@ jest.mock('@openshift/dynamic-plugin-sdk-utils', () => ({
 const k8sCreateResourceMock = k8sCreateResource as jest.Mock;
 
 describe('Add Workspace modal', () => {
-  let container: HTMLElement;
-
   beforeEach(() => {
     jest.resetModules();
     k8sCreateResourceMock.mockClear();
-    container = render(<WorkspaceAddButton workspaces={workspacesMockData} />).container;
+    render(<WorkspaceAddButton workspaces={workspacesMockData} />);
   });
```

Comment:
FYI - changes to the WorkspaceAddButton tests are to resolve an issue with some of the tests being evergreen.  This bug was found while working on the WorkspaceDeleteModal tets